### ISSUE TITLE
[Feature request]  rack-fx vst plugin as win32-dll

### ISSUE THREAD

**Issue by Z3R0C** [falkTX/Carla#757]
...over this way, you could get lv2 plugins into the LMMS mIxer. it's a request for all lmms@linux users. :)

**Comment by falkTX** [comment]
Latest develop branch already has this, by running `make plugin-wine` after a normal build.
The VST plugin generated is a shell though, which I am not sure if LMMS supports.

For LMMS it would be better anyway to have carla as fx and not just synth, so we would avoid getting wine involved for no reason.

**Comment by Z3R0C** [comment]
you're right. we need a Carla FX Plugin in LMMS. in the near future, LMMS will not support LV2. Carla is currently the only way.
